x-pack/plugins/security_solution/public/management/pages/policy/view/policy_forms/components/policy_form_layout.tsx

```diff
@@ -21,6 +20,7 @@ import { useDispatch } from 'react-redux';
 import { useLocation } from 'react-router-dom';
 import type { ApplicationStart } from '@kbn/core/public';
 import { toMountPoint } from '@kbn/kibana-react-plugin/public';
+import { KibanaPageTemplate } from '@kbn/shared-ux-page-kibana-template'
```

Comment:
Is there any difference if we import KibanaPageTemplate or EuiPageTemplate directly from @Eui?

Reply:
i'm not too sure? I assume it's the same thing